packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -135,19 +141,22 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
     }
 
     /**
-     * @custom:semver 1.2.0
+     * @custom:semver 1.3.0
      *
      * @param _l2Oracle                  Address of the L2OutputOracle contract.
      * @param _guardian                  Address that can pause deposits and withdrawals.
      * @param _paused                    Sets the contract's pausability state.
+     * @param _config                    Address of the SystemConfig contract.
      */
     constructor(
         L2OutputOracle _l2Oracle,
         address _guardian,
-        bool _paused
-    ) Semver(1, 2, 0) {
+        bool _paused,
+        SystemConfig _config
+    ) Semver(1, 3, 0) {
```

Comment:
Nit, semver was bumped [here](https://github.com/ethereum-optimism/optimism/pull/5006), which hasn't been deployed to prod yet. Do we need this bump?

Reply:
I believe we should specify bumping semver on each change because we other projects may deploy at any time. Kind of annoying, otherwise we should be very specific about releases of the contracts as a system

Reply:
OK, that makes sense.